include/pybind11/stl_bind.h

```diff
@@ -424,13 +424,18 @@ pybind11::class_<std::map<Key, T, Compare, Allocator>, holder_type> bind_map(pyb
 
     cl.def("__getitem__",
 		   [](Map &m, const KeyType &k) -> MappedType {
-			   if (m.count(k)) return m[k];
+			   if (m.count(k)) return m.at(k);
```

Comment:
The __delitem__ implementation can do the same thing as well, via `m.erase(it);`